libraries/ms-drivers/src/ads1259_adc.c

```diff
@@ -118,14 +116,8 @@ StatusCode ads1259_init(Ads1259Settings *settings, Ads1259Storage *storage) {
     .cs = settings->cs,
   };
   status_ok_or_return(spi_init(settings->spi_port, &spi_settings));
-  delay_us(100);
-  // first command that must be sent on power-up before registers can be read
-  prv_send_command(storage, ADS1259_STOP_READ_DATA_CONTINUOUS);
   status_ok_or_return(prv_configure_registers(storage));
-  prv_send_command(storage, ADS1259_OFFSET_CALIBRATION);
-  delay_ms(s_calibration_time_ms_lookup[ADS1259_DATA_RATE_SPS]);
-  prv_send_command(storage, ADS1259_GAIN_CALIBRATION);
-  delay_ms(s_calibration_time_ms_lookup[ADS1259_DATA_RATE_SPS]);
```

Comment:
Are the calibration commands not necessary? Also, removing these left `s_calibration_time_ms_lookup` unused, it should be removed as well.

Reply:
I'm taking hardware's word that the values were correct without calibration, so it's on them if they're not :) Removed `s_calibration_time_ms_lookup`